src/main/scala/com/cognite/spark/datasource/CdpRelation.scala

```diff
@@ -124,5 +125,20 @@ abstract class CdpRelation[T <: Product: Decoder](config: RelationConfig, shortN
   def delete(rows: Seq[Row]): IO[Unit] =
     throw new IllegalArgumentException(
       s"""$shortName does not support the "onconflict" option "delete".""")
+}
+
+trait InsertSchema {
```

Comment:
"Has" or "With" maybe?

Reply:
Yeah, was thinking the same thing, and if we going to have a prefix I'm in favor of `Has`. It's a trait to have InsertSchema